src/app/config/no-config/no-config.component.ts

```diff
@@ -10,27 +12,40 @@ import { Config, ConfigService } from '../config.service';
   styleUrls: ['./no-config.component.scss'],
 })
 export class NoConfigComponent implements OnInit {
-  public page = 0;
-  public totalPages = 6;
+  public page: number = 0;
+  public totalPages: number = 7;
 
   public configUpdate: boolean;
   public config: Config;
   public configErrors: string[];
   public configValid: boolean;
   public configSaved: string;
 
-  public octoprintConnection: boolean;
+  public objectvalues = Object.values;
+  public octoprintNodes: any = {
+    'other': {
+      'display': 'Other (> 1.4.0)',
```

Comment:
I think we can be more inclusive here. I would propose >1.3.5.

Reply:
sure I had no idea as I haven't finished going through everything, since we plan to use push when was that added?

Reply:
looks like GitHub botched up my comments.  This comment was for an other one.

For this one I mentioned something about when push was added to the api since we will be using that.  I still need to add a version compare lib like semver or something.  I can do that tonight too.